app/uk/gov/hmrc/individualsincomeapi/util/Dates.scala

```diff
@@ -42,6 +42,9 @@ object Dates extends ServicesConfig {
     if (fromTaxYear.startYr < TaxYear.current().startYr - selfAssessmentYearHistory)
```

Comment:
need to add error scenario in documentation

Reply:
👍